consensus/polybft/handlers.go

```diff
@@ -23,5 +26,5 @@ type PostEpochRequest struct {
 	SystemState SystemState
 
 	// ValidatorSet is the validator set for the new epoch
-	ValidatorSet *validatorSet
+	ValidatorSet validator.ValidatorSet
```

Comment:
Shouldn't this be a pointer?

Reply:
Here I switched from (now) unexported structure validatorSet (now in new validator package) to the ValidatorSet interface. This field of the PostEpochRequest struct is always used for read-only access, so it didn't need to be a pointer in the first place. Also, there are other places where this interface was already used (e.g. stateSyncManager, fsm) also without the pointer. The structure validatorSet itself implements interface as value-receiver, which also indicates it is intended to be used in read-only fashion.